src/main/java/com/hobbycollection/dao/CollectionDAO.java

```diff
@@ -13,23 +13,26 @@
 
     @Override
     public ArrayList<Collection> fetchAll() {
+
         return new ArrayList<>(allCollections.values());
     }
 
     @Override
     public Collection getItemByID(int id) {
+
         return allCollections.get(id);
     }
 
     @Override
     public Collection save(Collection collection) throws Exception {
-        Integer collectionID = collection.getID();
+        int collectionID = collection.getID();
```

Comment:
I changed variable type int instead of Integer.
As stated in the sources below, it is best practice to use "int" instead of "Integer" whenever possible because there
 is a difference between these variable types: "int is a primitive data type while Integer is a Wrapper class". 
int performs slightly better than Integer.
Sources:
* https://rhamedy.medium.com/a-short-summary-of-java-coding-best-practices-31283d0167d3
* https://www.geeksforgeeks.org/difference-between-an-integer-and-int-in-java/